packages/@lwc/engine-dom/src/apis/create-element.ts

```diff
@@ -105,26 +105,20 @@ export function createElement(
         );
     }
 
-    const element = document.createElement(sel);
```

Comment:
we can't just create the element via document.createElement anymore, we will rely on the upgradable or the existing registered element to avoid post creation upgrades